src/Discord.Net.Rest/Entities/Channels/ChannelHelper.cs

```diff
@@ -268,20 +268,17 @@ internal static class ChannelHelper
         public static async Task<RestUserMessage> SendMessageAsync(IMessageChannel channel, BaseDiscordClient client,
             string text, bool isTTS, Embed embed, AllowedMentions allowedMentions, MessageReference messageReference, MessageComponent component, ISticker[] stickers, RequestOptions options, Embed[] embeds)
         {
+            var listEmbeds = new List<Embed>();
```

Comment:
Cant this be simplifed with `??=` on `embeds` and just prepending the single embed to the array

Reply:
How would that work? Something like this?

```cs
embeds ??= Array.Empty<Embed>();
if (embed != null)
    embeds = new[] { embed }.Concat(embeds).ToArray();
```

Reply:
Almost, would be `embeds ??= embed != null ? new[] { embed } : new[0]`

Reply:
That doesn't prepend the single embed to the embed array, though.
Also `Array.Empty<Embed>()` is better than `new[0]` since it uses a static field instead of creating a new emtpy array every time.